onko_dicom.py

```diff
@@ -67,7 +68,18 @@ def open_file(self):
         if not full_path[0]:
             return
 
-        self.plot_w.plot_dcm(full_path[0])
+        try:
+            logging.log(logging.INFO, "Attempting to graph/open file")
+            self.plot_w.plot_dcm(full_path[0])
+            logging.log(logging.INFO, "successfully opened graph/file")
+
+        except pydicom.errors.InvalidDicomError as e:
+            logging.log(logging.ERROR, f"InvalidDicomError, Missing Dicom Header most likely. Error:({e})")
```

Comment:
most likely wording isnt the best

Reply:
Fair, It was mainly as i wasn't sure if other errors would give that as its hard to try and test only having certain bits. But i can change for header as i think thats a certain. Don't think its needed for this project so will remove the most likely